src/abaqus/Sketcher/ConstrainedSketch.py

```diff
@@ -480,7 +480,7 @@ def resetView(self):
         """This method resets the view to be perpendicular to the sketching plane."""
         pass
 
-    def rectangle(self, point1: tuple[float], point2: tuple[float]):
+    def rectangle(self, point1: tuple[float,float], point2: tuple[float,float]):
```

Comment:
There are a lot of incorrect errors just like it here, especially for arguments for coordinates, `tuple[float]` should be changed to `tuple[float]` (or maybe typing.Tuple[float, ...] for supporting Python <3.9) in a lot of other locations.

Reply:
Yes, I see. This correction was made from following the Tutorial lines. But I understand there is other places to correct.

Reply:
I fixed this in [abqpy (#47)](https://github.com/haiiliin/abqpy/pull/47).